src/typeahead/typeahead.js

```diff
@@ -295,6 +295,13 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position', 'ui.bootstrap
         }
       });
 
+      element.bind('focus', function (evt) {
+        if(minSearch == 0){
+          hasFocus = true;
+          getMatchesAsync(undefined);
```

Comment:
Actually, change undefined to an empty string for this line and the suggested one to add beforehand.